checkov/terraform/runner.py

```diff
@@ -101,6 +103,8 @@ def check_tf_definition(self, report, root_folder, runner_filter, collect_skip_c
 
     def run_all_blocks(self, definition, definitions_context, full_file_path, root_folder, report,
                        scanned_file, runner_filter, module_referrer: Optional[str]):
+        if not definition:
+            return
```

Comment:
Thanks for the quick fix. 

I think we would want a debug log in that case.

Reply:
Broke something. I'm away from my computer right now. I'll look at it when I get back.

Reply:
Looks like something broke in the recent `bc-python-hcl2` release. Error is:
> AttributeError("'Tree' object has no attribute 'items'")